docs/CODE_GUIDELINES.md

```diff
@@ -271,6 +271,30 @@ import { EdsThemeColorUtilitySuccessForeground } from 'src/tokens-dist/ts/colors
 <Icon color={EdsThemeColorUtilitySuccessForeground} />;
 ```
 
+- Use `:focus-visible` instead of `:focus`
```

Comment:
According to MDN, `focus-visible` isn't supported by our currently supported browsers: https://developer.mozilla.org/en-US/docs/Web/CSS/:focus-visible#browser_compatibility

So unfortunately, I don't think it makes sense to use this, yet.

We could use https://github.com/WICG/focus-visible in LP (like we do in Along). Not sure of the best way for EDS to tie into that, though.

Reply:
Woo, nice catch! So instead I should turn the `focus-visible` selectors into `focus`, right?

Reply:
I'm not sure. Do you mean going back to `:focus`, add https://github.com/WICG/focus-visible in LP, and it should just work? I think so 🤔

Reply:
> I'm not sure. Do you mean going back to `:focus`, add https://github.com/WICG/focus-visible in LP, and it should just work? I think so 🤔

Yes but also turn the usage of `focus-visible` in EDS to `focus` –  the Big Medium engineers were using `focus-visible`, so we have a mixture of focus implementation methods right now.

Reply:
Ah, gotcha. Yeah, changing any focus-visible-es back to focus makes sense 👍

Reply:
[PR for using :focus instead of :focus-visible](https://github.com/chanzuckerberg/edu-design-system/pull/1268)